diff.go

```diff
@@ -487,7 +487,9 @@ const (
 
 	DiffIgnoreWhitespace       DiffOptionsFlag = C.GIT_DIFF_IGNORE_WHITESPACE
 	DiffIgnoreWhitespaceChange DiffOptionsFlag = C.GIT_DIFF_IGNORE_WHITESPACE_CHANGE
-	DiffIgnoreWitespaceEol     DiffOptionsFlag = C.GIT_DIFF_IGNORE_WHITESPACE_EOL
+	DiffIgnoreWhitespaceEol    DiffOptionsFlag = C.GIT_DIFF_IGNORE_WHITESPACE_EOL
+	// Deprecated: use DiffIgnoreWhitespaceEol instead
+	DiffIgnoreWitespaceEol DiffOptionsFlag = C.GIT_DIFF_IGNORE_WHITESPACE_EOL
```

Comment:
I feel that the following looks more explicit that there is no difference between the two of them. What do you think?
```suggestion
	DiffIgnoreWitespaceEol = DiffIgnoreWhitespaceEol
```

Reply:
Agreed, I've updated the patch. :)

Reply:
Thank you.